doc/build-freebsd.md

```diff
@@ -52,20 +52,19 @@ git clone https://github.com/bitcoin/bitcoin.git
 ### 3. Install Optional Dependencies
 
 #### Wallet Dependencies
-It is not necessary to build wallet functionality to run bitcoind or the GUI. To enable legacy wallets, you must install `db5`. To enable [descriptor wallets](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md), `sqlite3` is required. Skip `db5` if you intend to *exclusively* use descriptor wallets
 
-###### Legacy Wallet Support
-`db5` is required to enable support for legacy wallets. Skip if you don't intend to use legacy wallets
+###### Wallet
 
-```bash
-pkg install db5
+`sqlite3` is used for the ([descriptor based](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md)) wallet.
```

Comment:
nit: `used for descriptor based wallets`

Reply:
Done in next push.